DOCUMENTATION.md

```diff
@@ -217,6 +219,153 @@ uint32_t secure_sum(uint32_t op1, uint32_t op2)
     return secure_gateway(my_box_name, __secure_sum, op1, op2)
 }
 ```
+---
+```C
+UVISOR_BOX_NAMESPACE(static const char const namespace[])
+```
+
+<table>
+  <tr>
+    <td>Description</td>
+    <td colspan="2"><p>Specify the namespace for a box.</p>
+
+      <p>The namespace of the box must be a null-terminated string no longer
```

Comment:
Not sure, but I'd say it should apply to all lines. The risk is that line breaks are interpreted as paragraph breaks, or that more in general we get inconsistent line breaks styles in exported docs.